code/modules/reagents/Chemistry-Reagents.dm

```diff
@@ -5118,6 +5118,216 @@
+				M.current.client.images += I
+				var/image/J = image('icons/mob/mob.dmi', loc = imagelocB, icon_state = "metaclub")
+				J.plane = REV_ANTAG_HUD_PLANE
+				new_buddy.current.client.images += J
+
+/datum/reagent/ethanol/waifu
+	name = "Waifu"
+	id = WAIFU
+	description = "Don't drink more than one waifu if you value your laifu."
+	color = "#664300"
+
+/datum/reagent/ethanol/waifu/on_mob_life(var/mob/living/M)
+	if(..())
+		return 1
+	if(M.gender == MALE)
+		M.setGender(FEMALE)
+	if(ishuman(M))
+		var/mob/living/carbon/human/H = M
+		if(!M.is_wearing_item(/obj/item/clothing/under/schoolgirl))
+			var/turf/T = get_turf(H)
+			T.turf_animation('icons/effects/96x96.dmi',"beamin",-32,0,MOB_LAYER+1,'sound/effects/rejuvinate.ogg',anim_plane = MOB_PLANE)
+			H.visible_message("<span class='warning'>[H] dons her magical girl outfit in a burst of light!</span>")
+			var/obj/item/clothing/under/schoolgirl/S = new /obj/item/clothing/under/schoolgirl(get_turf(H))
+			if(H.w_uniform)
+				H.u_equip(H.w_uniform, 1)
+			H.equip_to_slot(S, slot_w_uniform)
+			holder.remove_reagent(WAIFU,4) //Generating clothes costs extra reagent
+	M.regenerate_icons()
+
+/datum/reagent/ethanol/scientists_serendipity
+	name = "Scientist's Serendipity"
+	id = SCIENTISTS_SERENDIPITY
+	description = "Go ahead and blow the research budget on drinking this." //Can deconstruct a glass with this for loadsoftech
+	color = "#664300"
+	custom_metabolism = 0.01
+	dupeable = FALSE
+
+/datum/reagent/ethanol/beepskyclassic
+	name = "Beepsky Classic"
+	id = BEEPSKY_CLASSIC
+	description = "Some believe that the more modern Beepsky Smash was introduced to make this drink more popular."
+	color = "#664300" //rgb: 102, 67, 0
+	custom_metabolism = 2 //Ten times the normal rate.
+
+/datum/reagent/ethanol/beepskyclassic/on_mob_life(var/mob/living/M)
+	if(..())
+		return 1
+	if(ishuman(M))
+		var/mob/living/carbon/human/H = M
+		if(H.job in list("Security Officer", "Head of Security", "Detective", "Warden"))
+			playsound(get_turf(H), 'sound/voice/halt.ogg', 100, 1, 0)
+		else
+			H.Knockdown(10)
+			playsound(get_turf(H), 'sound/weapons/Egloves.ogg', 100, 1, -1)
+
+/datum/reagent/ethanol/spiders
+	name = "Spiders"
+	id = SPIDERS
+	description = "AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA."
+	color = "#666666" //rgb(102, 102, 102)
+	custom_metabolism = 0.01 //Spiders really 'hang around'
+
+/datum/reagent/ethanol/spiders/on_mob_life(var/mob/living/M)
+	if(..())
+		return 1
+	M.take_organ_damage(REM, 0) //Drinking a glass of live spiders is bad for you.
+	if(holder.get_reagent_amount(SPIDERS)>=4) //The main reason we need to have a minimum cost rather than just high custom metabolism is so that someone can't give themselves an IV of spiders for "fun"
+		var/calculate_offset = locate(M.x + pick(1,-1), M.y, M.z)
+		new /mob/living/simple_animal/hostile/giant_spider/spiderling(calculate_offset)
```

Comment:
Why does it only offset `x`?
Also, there's no check to make sure it doesn't put the spiderling on a wall, through a window, etc.

Reply:
Holy fucking wew this PR is running on code so old this got fixed in 2016
https://github.com/vgstation-coders/vgstation13/pull/12954

Reply:
Want to know a secret? Bees do this too when splitting swarms